src/components/FilterBar/FilterBar.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <div :class="$style['container']">
-    <p :class="$style['container__count']">{{ count }} items left</p>
+    <p :class="$style['container__count']">{{ activeCount }} items left</p>
     <ul :class="$style['container__filters']">
       <li>
         <button
+          @click="setFilter('all')"
           type="button"
           :class="
             currentFilter === 'all' ? $style['container__filters--active'] : ''
```

Comment:
Thank you for the link, it was an interesting read :) btw, the reason I used SCSS modules was because it was said so in the tasks, I didn't ponder about it much, just did what was written :D